src/meta/src/dashboard/mod.rs

```diff
@@ -351,7 +351,10 @@ pub(super) mod handlers {
 
         for worker_node in worker_nodes {
             let client = compute_clients.get(worker_node).await.map_err(err)?;
-            let result = client.stack_trace().await.map_err(err)?;
+            let result = client
+                .stack_trace(StackTraceRequest::default())
+                .await
+                .map_err(err)?;
```

Comment:
We may also allow specifying format here

Reply:
> It will be better to decouple the API for fetching the await-tree json, and the API to detect bottleneck actors.
That's because our bottleneck-detecting strategy may not be perfect. So we may wish to fetch the await-tree json from the user cluster first.

@kwannoel read my mind. I choose to write this in risectl is for decoupling dumping and analyzing. But of course we can allow one to specify format through meta's API.